pkg/storaged/dialog.jsx

```diff
@@ -692,26 +714,67 @@ export const CheckBox = (tag, title, options) => {
         initial_value: options.value || false,
 
         render: (val, change) => {
+            return <CheckBoxComponent tag={tag} val={val} title={title} options={options} update_function={change} />;
+        }
+    };
+};
+
+export const FieldSet = (tag, title, options) => {
```

Comment:
Let's call this `CheckBoxes` and make `type == "checkbox"` the default.  Checkboxes is all we have for now, and I think it's an important special case to give it its own name.